_integration-tests/tests/go-redis.v7/validation.json

```diff
@@ -24,8 +24,7 @@
             "span.kind": "client",
             "db.system": "redis",
             "redis.raw_command": "get test_key: ",
-            "out.host": "localhost",
-            "out.port": "6379"
+            "out.host": "localhost"
```

Comment:
Is it still needed? Testcontainers returns the address/port pair already

Reply:
This is making sure the correct tags are set on the trace... And I can't assert on the port because it's ephemeral.